bsyncpy/bsync.py

```diff
@@ -3149,64 +3197,83 @@ class EscalationRate(BSElement):
+        "Building Performance Institute (BPI): Advanced Home Energy Professional - Crew Leader (HEP-CL)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Energy Auditor (HEP-EA)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Quality Control Inspector (HEP-QCI)",
         "Building Performance Institute (BPI): Advanced Home Energy Professional - Retrofit Installer (HEP-RI)",
-        "Building Performance Institute (BPI): Advanced Home Energy Professional - Crew Leader (HEP-CL)",
+        "Building Performance Institute (BPI): Building Analyst (BA)",
         "Building Performance Institute (BPI): Multifamily Building Analyst",
-        "Residential Energy Services Network (RESNET) Certification",
-        "Residential Energy Services Network (RESNET) - Home Partner",
-        "Registered Architect (RA)",
-        "Refrigerating System Operating Engineer",
-        "High Pressure Boiler Operating Engineer",
         "Certified Commissioning Professional (CCP)",
-        "Associate Commissioning Professional (ACP)",
-        "Existing Building Commissioning Professional (EBCP)",
         "Commissioning Process Management Professional (CPMP)",
-        "Accredited Commissioning Process Authority Professional (CxAP)",
-        "NYSERDA FlexTech Consultant",
-        "ASHRAE Building Commissioning Professional (BCxP)",
-        "ASHRAE Building Energy Assessment Professional (BEAP)",
-        "ASHRAE Building Energy Modeling Professional (BEMP)",
         "Department of Buildings (DOB) Approved Agent",
-        "High-Performance Building Design Professional (HBDP)",
+        "Energy Management Association (EMA): Energy Management Professional (EMP)",
+        "Existing Building Commissioning Professional (EBCP)",
         "GreenPoint Rater Existing Home Multifamily Rater",
         "HERS Whole House (HERS II) Rater",
+        "High Pressure Boiler Operating Engineer",
+        "High-Performance Building Design Professional (HBDP)",
+        "International Facility Management Association (IFMA) Certified Facilities Manager (CFM)",
+        "International Facility Management Association (IFMA) Facility Management Professional (FMP)",
+        "International Facility Management Association (IFMA) Sustainability Facility Professional (SFP)",
         "International Union of Operating Engineers Certified Energy Specialist",
+        "MEP Professional Engineer",
+        "NYSERDA FlexTech Consultant",
+        "National Environmental Balancing Bureau (NEBB) Building Systems Commissioning (BSC)",
+        "National Environmental Balancing Bureau (NEBB) Commissioning Process Professional (CxPP)",
         "Northwest Energy Education Institute Energy Management Certification",
         "PhD in Mechanical Engineering",
+        "Professional Engineer (PE)",
+        "Refrigerating System Operating Engineer",
```

Comment:
Unsure whether this should have an acronym

Reply:
@ThibaultMarzullo, There does not appear to be one. Source: https://nyc-business.nyc.gov/nycbusiness/description/coq-q01